### ISSUE TITLE
[0.6.0-alpha.4]: Scales with key named "default" are broken

### ISSUE THREAD

**Issue by fcisio** [system-ui/theme-ui#1439]
**Describe the bug**
Installed `0.6.0-alpha.4` on Gatsby Starter, and it seems that the `boxShadows`, `borders` & `zIndices` keys are broken.

**To Reproduce**
[See the repro](https://github.com/fcisio/repro-theme-ui-broken-keys)

```json
"react": "^17.0.0",
"react-dom": "^17.0.0",
"theme-ui": "^0.6.0-alpha.4",
```

1. `yarn && yarn develop`
2. Inspect the `<h1 />` on the homepage

> You should see that the theme properties are working, with the exception of `box-shadow`, `border` & `z-index`

**Expected behavior**
All references should output the value set on the theme

**Screenshots**
![Shot_0 26](https://user-images.githubusercontent.com/46095852/104806927-03d03b80-57a9-11eb-9443-a3c971ea14f9.jpg)

**Additional context**
I extended the tests on a fork, but they seemed to all pass.

**Comment by lachlanjc** [comment]
Having this issue too. `borderRadius` is broken for me too

**Comment by hasparus** [comment]
I cherry-picked @fcisio's test on top of last 0.6.0-alpha.4 commit, and tests passed :/

So I made a tiny repro to see how it works on CodeSandbox, and it seems it's more complex than broken scales
https://codesandbox.io/s/lucid-cohen-sw28l?file=/src/App.js

**Comment by hasparus** [comment]
I just released 0.6.0-alpha.5 with a fix for this.

Proof by CodeSandbox:

Before: https://codesandbox.io/s/lucid-cohen-sw28l?file=/src/App.js
After: https://codesandbox.io/s/elastic-keller-uumrz?file=/src/App.js:230-276
